ghcide/src/Development/IDE/GHC/Orphans.hs

```diff
@@ -139,6 +143,11 @@ instance Show ModuleName where
 instance Hashable ModuleName where
     hashWithSalt salt = hashWithSalt salt . show
 
+instance Hashable Location
+instance Hashable Range
+instance Hashable Position
+instance Hashable UInt
+instance Hashable (Mod a) where hash n = hash (unMod n)
```

Comment:
Hmm, `GHC.Orphans` is not a good place since none of these types is defined in the `ghc` package.

Actually, it's probably better to revert to the previous state with the instances  in the rename plugin, and send PRs to haskell/lsp and `umod` with the missing instances so that we can remove the orphan instances in the future.  

Thank you!

Reply:
No problem, I've reverted the commit